src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -549,7 +549,7 @@ Copyright (c) .NET Foundation. All rights reserved.
 
       <!-- Copy satellite assemblies. -->
       <ResolvedFileToPublish Include="@(IntermediateSatelliteAssembliesWithTargetPath)">
-        <RelativePath>%(IntermediateSatelliteAssembliesWithTargetPath.Culture)\%(Filename)%(Extension)</RelativePath>
+        <RelativePath>%(IntermediateSatelliteAssembliesWithTargetPath.Culture)/%(Filename)%(Extension)</RelativePath>
```

Comment:
Will this work on windows? It seemed like the bundler was treating this as a string, causing `\` to be part of a filename on unix, so I naiively would have expected this to just move the problem to windows.

Reply:
AFAIK `/` is a reserved character in Windows and cannot be used for naming filenames/folders. This wouldn't be the only place where we use slashes for dir separators.

Reply:
Couldn't any other custom Target use backslash, and then it would be broken as well?

Reply:
Here's one example:

https://github.com/ashmind/Unbreakable/blob/a4cca1c7d1a21182485b12b3084cdaad01b0d7cb/Demo/Demo.csproj#L17-L54